lib/semian/circuit_breaker.rb

```diff
@@ -43,13 +49,16 @@ def mark_failed(error)
 
     def mark_success
       return unless half_open?
-      @successes += 1
+      #@successes += 1
```

Comment:
Yes it's still clear. Especially if rewritten as `@shared_circuit_breaker_data.successes += 1`